src/engine/engine_common.c

```diff
@@ -577,9 +577,19 @@ static inline int ocf_prepare_clines_miss(struct ocf_request *req)
 	return ocf_prepare_clines_evict(req);
 }
 
+static inline bool _defer_req(struct ocf_request *req)
+{
+	if (!ocf_req_test_clean_eviction(req))
+		return false;
+
+	if (req->byte_length % 65536 == 0)
```

Comment:
Can you use `ocf_cache_line_size_max` here? IMO it will look better and the value won't be hardcoded.

Reply:
`ocf_cache_line_size_max` is not directly related to the request size so I feel it wouldn't be appropriate to use it here, however I've added a different const